_Sidebar.md

```diff
@@ -15,6 +15,7 @@
 **Developing Oppia**
   * [[FAQs|Frequently-Asked-Questions]]
   * [[Installing Oppia|https://github.com/oppia/oppia/wiki/Installing-Oppia]]
+  * [[Upgrading python interpreter|Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory]]
```

Comment:
Maybe make the page name "Upgrading python version to 3.8.12"

Reply:
done